edgelet/edgelet-utils/src/lib.rs

```diff
@@ -90,7 +87,9 @@ pub fn append_dns_san_entries(sans: &str, names: &[&str]) -> String {
     let mut dns_sans = names
```

Comment:
nit: this should be renamed as it can be dns and ips

Reply:
Fixed